src/interface.rs

```diff
@@ -611,7 +661,12 @@ impl Interface {
                     routes6.push(match route.try_into() {
                         Ok(route) => route,
                         Err(e) => {
-                            log::warn!("{e}");
+                            log::warn!(
+                                "Failed parsing in {} for {}: {}",
+                                stringify!(ipv6_static.routes),
+                                self.name,
+                                e
+                            );
```

Comment:
My suggestion would be
```rust
                            log::warn!(
                                "Failed parsing route for {}: {}",
                                self.name,
                                e
                            );
```
and also the same for v4. (at the very least they should be the same)

Reply:
Okay fine, but then yea like I said at least the same message for v4 please